app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -122,36 +171,61 @@ private fun MyCoins(coins: List<CoinItemUiModel>) {
                 .padding(end = Dp16)
         )
 
-        LazyRow(
-            modifier = Modifier
-                .constrainAs(myCoins) {
-                    top.linkTo(myCoinsTitle.bottom, margin = Dp16)
-                    start.linkTo(parent.start)
-                },
-            contentPadding = PaddingValues(horizontal = Dp16),
-            horizontalArrangement = Arrangement.spacedBy(Dp16)
-        ) {
-            items(coins) { coin ->
-                CoinItem(coin)
+        if (showMyCoinsLoading) {
+            CircularProgressIndicator(
+                modifier = Modifier
+                    .constrainAs(myCoins) {
+                        top.linkTo(myCoinsTitle.bottom, margin = Dp16)
+                        start.linkTo(parent.start)
+                        end.linkTo(parent.end)
```

Comment:
Optional: Group these scopes by using `linkTo`.
```suggestion
                        linkTo(start = parent.start, end = parent.end)
```